common/performance_analysis/performance_plots.py

```diff
@@ -24,13 +24,28 @@
 import os
 from collections import defaultdict
 
-# Define the deployment types and their respective CSV files
+# Define the deployment types and their respective CSV files and colors
 deployment_types = {
-    'Single Node 4 Core': 'single_node_4_core.csv',
-    'Two Node 2 Core': 'two_node_2_core.csv',
-    'Two Node 4 Core': 'two_node_4_core.csv',
-    'Three Node 4 Core': 'three_node_4_core.csv',
-    'Four Node 4 Core': 'four_node_4_core.csv',
+    'Single Node 4 Core': {
+        'csv_file': 'single_node_4_core.csv',
+        'color': 'blue'
+    },
+    'Two Node 2 Core': {
+        'csv_file': 'two_node_2_core.csv',
+        'color': 'black'
```

Comment:
Shall we not use black as it's also used for other labels and lines in the graph?